src/Compilers/CSharp/Portable/Lowering/IteratorRewriter/IteratorRewriter.cs

```diff
@@ -174,10 +174,7 @@ protected override void GenerateControlFields()
 
             // if it is an enumerable, and either Environment.CurrentManagedThreadId or Thread.ManagedThreadId are available
             // add a field: int initialThreadId
-            bool addInitialThreadId =
```

Comment:
:memo: both changes in this file just refactoring (extract method) in preparation for later PR related to thread ID.